pages/search.js

```diff
@@ -44,9 +45,16 @@ export default function Search({ users }) {
 
     if (value.length >= 3) {
       setThreeOrMore(true);
-      results = users.filter((user) =>
-        user.name.toLowerCase().includes(value.toLowerCase())
-      );
+      results = users.filter((user) => {
+        if (user.name.toLowerCase().includes(value.toLowerCase())) return true;
```

Comment:
Similar to line 54
```suggestion
        if (user.name.toLowerCase().includes(value.toLowerCase())) {
           return true;
        }
```

Reply:
Sorry @eddiejaoude this mistake will not be repeated